src/Symfony/Component/Serializer/Exception/AggregatedException.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Exception;
+
+final class AggregatedException extends RuntimeException implements AggregableExceptionInterface
+{
+    /**
+     * @var bool
+     */
+    public $has = false;
+
+    /**
+     * @var array
+     */
+    private $collection;
+
+    public function __construct(\Throwable $previous = null)
+    {
+        parent::__construct('Errors occurred during the denormalization process', 0, $previous);
+    }
+
+    public function add(string $param, AggregableExceptionInterface $exception): self
```

Comment:
I think it should be renamed to `set`:
```suggestion
    public function set(string $param, AggregableExceptionInterface $exception): self
```
Usually `add` methods don't care about the key.

Reply:
Maybe `put` would be a better choice than `set` if we speak in terms of collection?

Reply:
replace `add` to `put`

Reply:
https://symfony.com/doc/current/contributing/code/conventions.html